src/operator/windowToggle.ts

```diff
@@ -131,10 +131,16 @@ class WindowToggleSubscriber<T, O> extends OuterSubscriber<T, any> {
         const context = { window, subscription };
         this.contexts.push(context);
         const innerSubscription = subscribeToResult(this, closingNotifier, context);
```

Comment:
What about this inner subscription to the empty Observable, when is it unsubscribed?